docs/source/Contribs/Components.md

```diff
@@ -0,0 +1,156 @@
+# Components
```

Comment:
Ehm :point_right::point_left: Since this was started (I guess it would be _Contrib by ChrisLR 2022_ now...) the way contribs are documented has changed. The documentation is to be kept in a `README.md` file in the contrib's main folder instead. The doc-building mechanism will extract the README into a doc page (so it needs to be valid markdown) and place it in the right place in the docs (and also include it in a nice index of contribs). 

So please add this as `evennia/contrib/base_systems/components/README.md` instead.